libraries/SITL/SIM_Precland.cpp

```diff
@@ -198,3 +205,7 @@ void SIM_Precland::set_default_location(float lat, float lon, int16_t yaw) {
         _orient_yaw.set(yaw);
     }
 }
+
+bool SIM_Precland::enable_target_distace() const {
+    return _options & 1;
+}
```

Comment:
```suggestion
```suggestion
bool SIM_Precland::option_enabled(Option option) const {
    return (_options & uint8_t(option)) != 0;
}
```

and
```
enum Option {
    ENABLE_TARGET_DISTANCE=1,
};
```

... and other changes to match.  This matches the pattern we use in ArduPilot for similar options.

Reply:
Done! Please check.